astrokat/observe_main.py

```diff
@@ -413,6 +416,15 @@ def run_observation(opts, kat):
             description = opts.proposal_description
         session_opts["description"] = description
 
+    if "adjust_pointing" not in vars(opts):
+        session_opts = vars(opts)
+        adjust_pointing = obs_plan_params["adjust_pointing"]
+        max_age = obs_plan_params["max_age"]
```

Comment:
These two also come out of the "reference_pointing_scan" section.
I would make them optional, only setting them if they are there.